api_tests/comments/views/test_comment_detail.py

```diff
@@ -147,6 +148,7 @@ def test_comment_has_node_link(self):
         assert_equal(urlparse(url).path, expected_url)
 
     def test_registration_comment_has_node_link(self):
+        import ipdb;ipdb.set_trace()
```

Comment:
@sloria This is the actual test that's failing with, well, before I broke it all, something along the lines of `'/v2/nodes/123ab/' != '/v2/registrations/123ab/'`.